Makefile

```diff
@@ -28,20 +30,20 @@ clean:
 	@find . -name '__pycache__' -delete
 	@find . -name '*egg-info' -delete
 
-.PHONY: install
+.PHONY: setup
 install:
-	@pip install --user -U pip
-	@pip install --user "Cython<3"
-	@pip install --user -e .
```

Comment:
Re-add line 35 (`@pip install --user -e .`) to the Makefile so that `make tests` can be ran in the JupyterLab terminal (or anywhere after `make install` has been ran. So far the `asl` lib is installed only in the asl_kernel.

Reply:
Added `python -m` and solved the issue without reinstalling the dependencies twice.
https://github.com/GoogleCloudPlatform/asl-ml-immersion/pull/465/commits/fe577ba3af116f85ac3ca9a9302380d0e314af3e